cicd/3-app/load-test.template.yml

```diff
@@ -0,0 +1,87 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: The template used to create the required services for load testing
+Parameters:
+  LoadTestImage:
+    Type: String
+    Description: Load testing docker image. This should be the full name of the image.
+Resources:
+  ECSCluster:
+    Type: 'AWS::ECS::Cluster'
+    Properties:
+      CapacityProviders:
+        - FARGATE
+        - FARGATE_SPOT
+      ClusterName: !Sub "${AWS::StackName}-cluster"
+      ClusterSettings:
+        - Name: containerInsights
+          Value: disabled
+      Configuration:
+        ExecuteCommandConfiguration:
+          Logging: DEFAULT
+      Tags:
+        - Key: "javabuilder-mode"
+          Value: "load-test"
+  CloudwatchLogsGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "ECSLogGroup-${AWS::StackName}"
+      RetentionInDays: 14
+  ECSTaskDefinition:
+    Type: AWS::ECS::TaskDefinition
+    DependsOn:
+      - ECSCluster
+    Properties: 
+      Family: !Sub "${AWS::StackName}-task"
+      ContainerDefinitions: 
+        - Name: javabuilder-load-test
+          Image: !Ref LoadTestImage
+          Cpu: 0
+          Memory: 512
+          Environment:
+            - Name: PRIVATE_KEY
+              Value: arn:aws:secretsmanager:us-east-1:475661607190:secret:production/cdo/javabuilder_load_test_key-O3hwTg
+            - Name: PRIVATE_KEY_PASSWORD
+              Value: arn:aws:secretsmanager:us-east-1:475661607190:secret:production/cdo/javabuilder_load_test_key_password-MFH2XA
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group:  !Ref CloudwatchLogsGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix:  !Sub "${AWS::StackName}-task"
+      Cpu: 256
+      ExecutionRoleArn: !Ref ExecutionRole
+      Memory: 512
+      NetworkMode: awsvpc
+      RuntimePlatform: 
+        OperatingSystemFamily: LINUX
+      Tags: 
+        - Key: "javabuilder-mode"
+          Value: "load-test"
```

Comment:
We can add tags to the `cloudformation deploy` step and it will apply the same set of tags to all resources in the stack.

Reply:
done!